libnopegl/src/pgcraft.c

```diff
@@ -1231,26 +1234,33 @@ static int32_t get_texture_index(const struct pgcraft *s, const char *name)
     return -1;
 }
 
-static void probe_texture_info_elems(const struct pgcraft *s, struct pgcraft_texture_info_field *fields)
+static void probe_texture_info_elems(const struct pgcraft *s,
+                                     struct pgcraft_texture *texture,
```

Comment:
Again here the added `texture` can be `const`.

Reply:
Done.